src/script/page/AccentColorPicker.tsx

```diff
@@ -81,51 +71,63 @@ const AccentColorPicker: React.FunctionComponent<AccentColorPickerProps> = ({use
                   data-uie-name="do-set-accent-color"
                   data-uie-value={id}
                   css={{
-                    '& + span': {
+                    '& + label > span:first-child': {
                       color: color,
                       cursor: 'pointer',
                       display: 'inline-block',
                       position: 'relative',
                     },
-                    '& + span::after': {
+                    '& + label > span:first-child::after': {
                       ...CSS_SQUARE(10),
                       background: 'currentColor',
                       left: '-5px',
                       top: '-5px',
                     },
-                    '& + span::before': {
+                    '& + label > span:first-child::before': {
                       ...CSS_SQUARE(16),
                       left: '-8px',
                       top: '-8px',
                     },
-                    '& + span::before, & + span::after': {
+                    '& + label > span:first-child::before, & + label > span:first-child::after': {
                       borderRadius: '50%',
                       content: '""',
                       display: 'inline-block',
                       position: 'absolute',
                       transition: 'all 0.15s ease-out',
                     },
-                    '&:checked + span::after': {
+                    '&:checked + label > span:first-child::after': {
                       left: '-5px',
                       top: '-5px',
                     },
-                    '&:checked + span::before': {
+                    '&:checked + label > span:first-child::before': {
                       border: '1px solid currentColor',
                     },
-                    '&:focus + span::before': {
+                    '&:focus + label > span:first-child::before': {
                       ...CSS_SQUARE(16),
                       outline: '1px solid Highlight',
                     },
                     opacity: 0,
                   }}
                 />
-                <span onClick={() => doSetAccentColor(id)} />
                 <label
                   htmlFor={String(id)}
-                  onClick={() => doSetAccentColor(id)}
-                  style={{cursor: 'pointer', fontSize: '11px', marginTop: '14px', textTransform: 'capitalize'}}
+                  style={{
+                    alignItems: 'center',
+                    cursor: 'pointer',
+                    display: 'flex',
+                    flexDirection: 'column',
+                  }}
                 >
-                  {name}
+                  <span />
```

Comment:
Yeah, it's a bit of a mess these days. I think in the near future it would be nice that we use colocated css style right next to the component using a `Component.css` or `Component.style.ts` or something like this...
If you have a suggestion, I am all ears for this